pkg/ebpf/tracee.go

```diff
@@ -18,6 +17,8 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/aquasecurity/tracee/pkg/events/queue"
```

Comment:
If you move this, let's put it below just before "github.com/aquasecurity/tracee/pkg/events/sorting"

Reply:
I think this moved due to automatic formatting, I didn't touch it myself.

Reply:
done